### PR TITLE
2022 update

### PR DESCRIPTION
**Pull request by jonathanbossenger**

### CODE REVIEW

**Comment by jonathanbossenger** [comment]
@ianmjones as part of the npm scripts article update you are TE on, I've needed to update the packages in the package.json.

Would you mind reviewing this PR to let me know if you spot any obvious issues?

Thanks

File: package.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
     "concat:css": "concat -o dist/css/styles.css dist/css/style.css assets/css/test.css",
     "concat:js": "mkdir -p dist/js && concat -o dist/js/scripts.js assets/js/test1.js assets/js/test2.js",
     "concat": "npm run concat:css && npm run concat:js",
     "cssmin": "cleancss -o dist/css/styles.min.css dist/css/styles.css",
     "uglify": "uglifyjs -o dist/js/scripts.min.js dist/js/scripts.js",
-    "imagemin": "imagemin --out-dir=dist/img assets/img/**/*.{png,jpg,gif}",
+    "squoosh": "squoosh-cli --mozjpeg '{quality:70}' assets/img -d dist/img",
     "build:css": "npm run scss && npm run concat:css && npm run cssmin",
     "build:js": "npm run concat:js && npm run uglify",
     "build:img": "npm run imagemin",
```
Comment (ianmjones) [inline package.json]: Should this be changed to `npm run squoosh`?

**Review by ianmjones — CHANGES_REQUESTED** [review]
@jonathanbossenger unfortunately the `build` script fails.

2022-01-19-125724_1743x2160_scrot

I've left a code comment where I think the problem might be.

**Review by ianmjones — APPROVED** [review]
LGTM :+1:

File: package.json (hunk 1 above)
Comment (jonathanbossenger) [reply package.json]: Thanks, I picked this up as well.

**Pull request merged**